mono/utils/atomic.h

```diff
@@ -464,4 +464,12 @@ extern void InterlockedWritePointer(volatile gpointer *dst, gpointer val);
 #define InterlockedAddP(p,add) InterlockedAdd64 ((volatile gint64*)p, (gint64)add)
 #endif
 
+/* The following functions cannot be found on any platform, and thus they can be declared without further existence checks */
+
+static inline gint32
+InterlockedSubtract (volatile gint32 *dest, gint32 sub)
+{
```

Comment:
@BrzVlad @alexrp is it OK to put this function there? Also, is it OK to use `InterlockedAdd ()` with `-1` or should I use `gcc_sync_add_and_fetch ()`, `gcc_sync_sub_and_fetch ()`, ... to build it?